data/wanderer/wanderers start.txt

```diff
@@ -36,8 +36,7 @@ mission "First Contact: Wanderer (Before Hai)"
 	source
 		attributes "wanderer"
 	to offer
-		not "First Contact: Hai: offered"
-		not "First Contact: Unfettered: offered"
+		not "event: hai-human resolution announced"
```

Comment:
Unless this mission is failed somewhere, this will now mean that anyone who never got this mission but has already started the Wanderer story but hasn't finished HR (say, because they just updated) will get this mission, even though it makes no sense.